gswidgetkit/scrollbar.py

```diff
@@ -13,3 +13,431 @@
+            self.SetSizer(sizer)
+
+            # # SuperScrolledWindow
+            self.ssw = SuperScrolledWindow(parent=self)
+            sizer.Add(self.ssw, proportion=1, flag=wx.EXPAND | wx.ALL, border=10)
+
+            # Get ScrolledPanel and add stuff
+            self.panel = self.ssw.get_panel()
+            self.panel.SetBackgroundColour("#FFFFFF")
+            self.panel.SetForegroundColour("#000000")
+            self.main_sizer = wx.GridBagSizer(vgap=5, hgap=5)
+            self.panel.SetSizer(self.main_sizer)
+
+            self.init_panel()
+
+            # Setup up scrolling
+            self.ssw.setup(scroll_x=True, scroll_y=True, rate_x=20, rate_y=20, scroll_to_top=True,
+                           scroll_into_view=False)
+
+            # Add a pointless button on the frame
+            self._mode = 'light'
+            self._btn_mode = wx.Button(parent=self, label='Dark Mode', name='mode')
+            sizer.Add(self._btn_mode, flag=wx.ALL, border=10)
+
+            self.Layout()
+            self.Fit()
+            self.SetInitialSize((300, 300))
+
+            self.Bind(event=wx.EVT_BUTTON, handler=self.__on_button)
+
+        def __on_button(self, event):
+            if event.EventObject.GetName() == "scroll_me":
+                self.ssw.scroll(x=20, y=20)
+
+            elif event.EventObject.GetName() == 'mode':
+                if self._mode == 'light':
+                    self._btn_mode.SetLabel(label='Light Mode')
+                    self._mode = 'dark'
+                    self.ssw.set_colors(bg="#212121", fg="#FFFFFF")
+                else:
+                    self._btn_mode.SetLabel(label='Dark Mode')
+                    self._mode = 'light'
+                    self.ssw.set_colors(bg="#FFFFFF", fg="#000000")
+
+            event.Skip()
+
+        def init_panel(self):
+            """
+            Add stuff...
+            """
+            text1 = wx.StaticText(parent=self.panel, label="words words words. these are words. blah blah blah "
+                                                           "this is a long statictext to take up room. words words "
+                                                           "words. these are words. i know words. these are words")
+            text1.Wrap(200)
+            self.main_sizer.Add(text1, pos=(0, 0), flag=wx.ALL, border=5)
+
+            btn1 = wx.Button(parent=self.panel, label="Scroll Me", name="scroll_me")
+            self.main_sizer.Add(btn1, pos=(0, 1), flag=wx.ALL, border=5)
+
+            for x in range(4):
+                for y in range(4):
+                    p = wx.Panel(parent=self.panel, size=(100, 100))
+                    p.SetBackgroundColour("#85e97a")
+                    self.main_sizer.Add(p, pos=(x+1, y+1), flag=wx.ALL, border=5)
+
+            self.panel.Layout()
+
+    app = wx.App(False)
+    frame = TestFrame()
+
+    frame.Show()
+
+    app.MainLoop()
+    app.ExitMainLoop()
+    app.Destroy()
```

Comment:
Move the demo to ``demo.py`` (so it is consistent with all widgets in this repository)